postgres-appliance/build_scripts/patroni_wale.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+## -------------------------
+## Install patroni and wal-e
+## -------------------------
+
+export DEBIAN_FRONTEND=noninteractive
+
+set -ex
+
+BUILD_PACKAGES=(python3-pip python3-wheel python3-dev git patchutils binutils gcc)
+
+apt-get update
+
+# install most of the patroni dependencies from ubuntu packages
+apt-cache depends patroni \
+        | sed -n -e 's/.* Depends:(python3-.\+\)$/\1/p' \
+        | grep -Ev '^python3-(sphinx|etcd|consul|kazoo|kubernetes)' \
+        | xargs apt-get install -y "${BUILD_PACKAGES[@]}" python3-pystache python3-requests
+
+pip3 install setuptools
+
+if [ "$DEMO" != "true" ]; then
+    export EXTRAS=",etcd,consul,zookeeper,aws"
```

Comment:
here we don't need to export
```suggestion
    EXTRAS=",etcd,consul,zookeeper,aws"
```